bin/postman-test/collections/core-metadata.postman_collection.json

```diff
@@ -9388,7 +9656,544 @@
 								"provisionwatcher"
 							]
 						},
-						"description": "Add a new ProvisionWatcher - name must be unique. Returns ServiceException (HTTP 503) for unknown or unanticipated issues.  Returns DataValidationException (HTTP 409) if profile service are unknown"
+						"description": "Attempt to update a provision watcher using an invalid request body (in this case, an addressable)."
+					},
+					"response": []
+				},
+				{
+					"name": "316 http://localhost:48081/api/v1/provisionwatcher",
```

Comment:
I got `404 Not Found` for this testcase.

Reply:
Can you ensure you have the latest from master on edgex-go? I ran these again this morning and I'm seeing the 409, as expected.

Reply:
@brandonforster Ginny might test it before the latest docker images ready.  we will check again tomorrow.

Reply:
I still got `404 Not Found`.

I used the following steps:
1. `bash deploy-edgeX.sh`
2.  `bin/env.sh`
3. `bash bin/run.sh -md`

Reply:
Found the issue, I had some lingering test data in my local instance (not Docker.) Should work now, thanks for the catch!

Reply:
It returns the correct status code now.